### DISCUSSION TITLE
Documentation discussions

### DISCUSSION THREAD

**Discussion by shakthinandana** [Ashwinshankar98/TeachersPetBot#23] [General]
Hey Guys,
Listing all the required documentations.

1. Phase1 vs Phase2
2. Readme update with new features added
3. docs generation
4. Video editing with new features
5. function descriptions in isolation
6. Rubric for Phase 2

If I've missed anything, please let me know here and add an issue if needed :)

**Comment by shakthinandana** [comment]
Any changes in Contributing .md ? @all
Check Code of Conduct also?

**Reply by ithaaswin** [reply]
Checked Contributing, Readme, and code of conduct.
Everything looks good!!!

**Comment by shakthinandana** [comment]
All updates done to requirements file and readme is updated with all path changes

**Comment by shakthinandana** [comment]
Ph1vPh2 doc is updated, along with the rubric file

**Reply by Ashwinshankar98** [reply]
Looks good

**Comment by shakthinandana** [comment]
File in isolation created for function details and description.

**Comment by shakthinandana** [comment]
Readme is updated with the new features and a video about the new features has been added

**Reply by Ashwinshankar98** [reply]
verified.
